geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa

```diff
@@ -33,17 +33,8 @@ if [[ -n "$HADOOP_HOME" ]]; then
   fi
 fi
 
-# Warn user about missing environment variables
-if [[ (-z "$ACCUMULO_HOME") || (-z "$HADOOP_HOME") ]]; then
-  message="Warning: you have not set ACCUMULO_HOME and/or HADOOP_HOME as environment variables."
-  message="${message}\nGeoMesa tools will not run without the appropriate Accumulo and Hadoop jars in the tools classpath."
-  message="${message}\nPlease ensure that those jars are present in the classpath by running 'geomesa classpath' ."
-  message="${message}\nTo take corrective action, please place the necessary jar files in the lib directory of geomesa-tools."
-  echo >&2 ""
-  echo -e >&2 "$message"
-  echo >&2 ""
-elif [[ (-z "$ACCUMULO_CONF_DIR") || (-z "$HADOOP_CONF_DIR") ]]; then
-  message="Warning: you have not set ACCUMULO_CONF_DIR and/or HADOOP_CONF_DIR as environment variables."
+if [[ (-z "$ACCUMULO_CONF_DIR") || (-z "$HADOOP_CONF_DIR") ]]; then
```

Comment:
I've just removed the Accumulo conf dir warning. The hadoop one should remain, correct?